Vue2LeafletVectorGridProtobuf.vue

```diff
@@ -3,20 +3,6 @@
 <script>
 import L from 'leaflet'
 import 'leaflet.vectorgrid'
-import v2l from 'vue2-leaflet'
-
-if (typeof(L) === "undefined") {
-    throw new Error("leaflet library must be installed in order " +
-                    "to use vue2-leaflet-vectorgrid.")
-}
-if (typeof(L.vectorGrid) === "undefined") {
-    throw new Error("leaflet.vectorgrid library must be installed " +
-                    "in order to use vue2-leaflet-vectorgrid.")
-}
-if (typeof(v2l) === "undefined") {
-    throw new Error("vue2-leaflet Vue Component must be installed " +
-                    "in order to use vue2-leaflet-vectorgrid.")
-}
 
```

Comment:
Why are you not leaving those in? Does it not make sense to test for the installation of the dependencies?

Reply:
Two reasons:
1. npm show if some `peerDependencies` not installed.
2. In `vue2-leaflet` after version 2 not have default import. Now imports only needed components.
We may stay this checks if import { LMap }.